lib/cn/csched_sp3_work.c

```diff
@@ -495,6 +493,8 @@ sp3_work_joinable(struct cn_tree_node *right, const struct sp3_thresholds *thres
     if (accum * 100 > thresh->lcomp_split_keys * pct)
         return NULL;
 
+    ev_debug(1);
+    return NULL; // remove before flight
```

Comment:
Remove this?

Reply:
Yep, but only if your patch goes in first..